src/app/dashboard/oracle/page.tsx

```
@@ -0,0 +1,154 @@
+  }, [query]);
+
+  useEffect(() => {
+    const loadPrice = async () => {
+      if (!lastTs) return;
+      try {
+        const result = await query("price", [
+          nativeToScVal({ Symbol: oracleAssetSymbol }),
+          nativeToScVal(BigInt(lastTs)),
+        ]);
+        if (result && typeof result === "object" && "price" in result) {
+          const p = (result as any).price as bigint;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)